tests/acceptance/cleanup-test.js

```diff
@@ -0,0 +1,35 @@
+import { test } from 'qunit';
+import moduleForAcceptance from '../../tests/helpers/module-for-acceptance';
+
+import { selectorToExist, cleanup, activeCount } from 'ember-wait-for-test-helper/wait-for';
+
+moduleForAcceptance('Acceptance | cleanup');
+
+test('I should be able to cancel waiters', function(assert) {
+  assert.expect(2);
+
+  visit('/');
+
+  // this should cause our test to freeze until the runner
+  // times out. at the point the suite will move on to the
+  // next test, so we want to make sure we cancel this timer.
+  waitFor(selectorToExist('.this-will-never-exist'));
+
```

Comment:
Don't think so. Any assertion would need to be inside of an `andThen` since waitFor is an async helper. Best I could come up with was assertion in setTimeout. However, I added an assert 0 active check before the test starts, so if theres another bad test in the suite it wont leak here.